main.rb

```diff
@@ -1,44 +1,54 @@
 require_relative 'app'
+require_relative 'user_input_helper'
 
-def render_choices
-  puts 'Choose your Option:'
-  puts '1. List all books.'
-  puts '2. List all people.'
-  puts '3. Create a person.'
-  puts '4. Create a book.'
-  puts '5. Create a rental.'
-  puts '6. List all rentals for a given person id.'
-  puts '7. Quit'
-end
+class Main
+  includes UserInputHelper
```

Comment:
- [ ] Great effort working on the changes requested by the previous reviewer, I must really commend the effort. But I realized I could not start the app using the `ruby main.rb` command and I notice what happens to be the factor behind that is the you have included your `UserInputHelper`. it should be `include` and not `includes`.

Screenshot 👇 

![main](https://github.com/ANTOINE1128/OOP--School--Library/assets/78634841/c0640417-4c1b-4d47-b529-3ef1c46f3b7e)